content/docs/command-reference/dag.md

```diff
@@ -1,49 +1,49 @@
 # dag
 
-Show [stages](/doc/command-reference/run) in a pipeline that lead to the
-specified stage. By default it lists
-[DVC-files](/doc/user-guide/dvc-files-and-directories).
+Display connected [stages](/doc/command-reference/run) in the pipelines that
+contain them.
 
 ## Synopsis
 
 ```usage
 usage: dvc dag [-h] [-q | -v] [--dot] [--full] [target]
 
 positional arguments:
-  targets         Stage or output to show pipeline for (optional)
-                  Finds all stages in the workspace by default.
+  target          Stage or output to show pipeline for (optional)
+                  Uses all stages in the workspace by default.
 ```
 
 ## Description
 
 A data pipeline, in general, is a series of data processing
 [stages](/doc/command-reference/run) (for example console commands that take an
 input and produce an <abbr>output</abbr>). A pipeline may produce intermediate
-data, and has a final result. Machine learning (ML) pipelines typically start a
-with large raw datasets, include intermediate featurization and training stages,
-and produce a final model, as well as accuracy
-[metrics](/doc/command-reference/metrics).
+data, and has a final result.
+
+Machine learning (ML) pipelines typically start a with large raw datasets,
```

Comment:
let's use Data processing or ML pipelines ...

Reply:
and done